crates/rolldown_plugin_module_preload_polyfill/src/module_preload_polyfill.js

```diff
@@ -0,0 +1,66 @@
+// Port from https://github.com/vitejs/vite/blob/main/packages/vite/src/node/plugins/modulePreloadPolyfill.ts#L34
```

Comment:
I think you could remove this line, since it would appear in runtime code, you could claim it in your plugin code.

Reply:
agree

Reply:
It doesn't seem to appear in the runtime code, and it seems to be declared so in other plugins as well?

https://github.com/rolldown/rolldown/blob/f5ffbf4238e0ddc97180b4c3e257396b109b42cf/packages/rolldown/tests/fixtures/builtin-plugin/module_preload_polyfill/main.snap#L1-L3

https://github.com/rolldown/rolldown/blob/f5ffbf4238e0ddc97180b4c3e257396b109b42cf/crates/rolldown_plugin_wasm/src/wasm_runtime.js#L1

https://github.com/rolldown/rolldown/blob/f5ffbf4238e0ddc97180b4c3e257396b109b42cf/crates/rolldown_plugin_dynamic_import_vars/src/dynamic_import_helper.js#L1

Reply:
Should I remove them together?

Reply:
I know why, because currently, oxc will not preserve all comments. ok, you could ignore that, we could remove it when necessary.